packages/SwingSet/src/kernel/vatManager/transcript.js

```diff
@@ -26,7 +26,13 @@ export function makeTranscriptManager(
     };
   }
 
+  const gcSyscalls = new Set(['dropImports', 'retireImports', 'retireExports']);
```

Comment:
I think I'd label this `unrecordedSyscalls` or something like that.  Today it's GC but conceivably in the future there could be other things that fall into this category.

Reply:
Good point, but I think it's slightly more likely to move a different direction: we don't record them during the initial execution, but we still collect something about them during replay (to deal with possible early `dropImports`). So "unrecorded" won't quite capture it.. the code on the `if X.include(syscallType)` side will want to recognize GC syscalls and do something special with them, record/not-record/collect. So my hunch is that we'll continue to mark this particular set as "GC syscalls", but the mapping (or not) of "GC syscall" to "don't record" will be made explicit at the point of use.